components/inputs/input.css

```diff
@@ -0,0 +1,45 @@
+/* Styling for input components group */
+.input-group {
+  margin: var(--scale-sm);
+  padding: var(--scale-sm);
+}
+
+/* Common Styling for input boxes */
+.input-group input[type="text"],
+input[type="password"],
+input[type="email"],
+input[type="tel"],
+input[type="search"],
+input[type="url"] {
+  background-color: var(--default-bg-color);
+  border: 1px solid var(--secondary-hover);
+  float: left;
+  margin: var(--scale-sm) 0;
+  outline: none;
```

Comment:
Try to avoid using outline:none, an alternative for this will be 
outline: 2px solid transparent;
outline-offset: 2px;

Reply:
OK.
Any reason why we should not use `outline:none;`

Reply:
Using outline: none  makes your site significantly less accessible to any keyboard only user, not only those with reduced vision.

You can read more here
https://www.a11yproject.com/posts/never-remove-css-outlines/#:~:text=Some%20browser%2Fscreen%20reader%20combinations,only%20those%20with%20reduced%20vision.